engine/src/main/java/org/terasology/rendering/dag/nodes/AlphaRejectBlocksNode.java

```diff
@@ -67,20 +71,22 @@
     private Material chunkShader;
     private SetWireframe wireframeStateChange;
     private RenderingDebugConfig renderingDebugConfig;
+    private FBO sceneOpaqueFbo;
 
     /**
      * Initialises the node. -Must- be called once after instantiation.
      */
     @Override
     public void initialise() {
         playerCamera = worldRenderer.getActiveCamera();
+        sceneOpaqueFbo = displayResolutionDependentFBOs.get(SCENE_OPAQUE);
 
         wireframeStateChange = new SetWireframe(true);
         renderingDebugConfig = config.getRendering().getDebug();
         new WireframeTrigger(renderingDebugConfig, this);
 
         addDesiredStateChange(new LookThrough(playerCamera));
-        addDesiredStateChange(new SetViewportToSizeOf(READ_ONLY_GBUFFER));
+        addDesiredStateChange(new SetViewportToSizeOf(SCENE_OPAQUE, displayResolutionDependentFBOs));
```

Comment:
Quite a lot of these lines (initializing FBOs) will be shifted to the update() method in the resizing fix PR, so we might have to shift lines again.. But I'll do as requested for now :)